tensorflow/core/platform/BUILD

```diff
@@ -1041,13 +1041,22 @@ cc_binary(
     ] + if_not_android([":rocm_rocdl_path"]),
 )
 
+alias (
+    name = "gpu_all_runtime",
+    actual = if_google(
+        "//tensorflow/compiler/xla/stream_executor/cuda:all_runtime",
+        "//tensorflow/compiler/xla/stream_executor/rocm:all_runtime"
+    ),
+    visibility = ["//visibility:private"],
+)
+
 # This do-nothing lib should only be used as a dep in cuda_deps in :stream_executor cuda lib because
 # the "+" operator doesn't work for bazel in stream_executor cuda_deps.
 cc_library(
     name = "private_static_dep",
     visibility = ["//visibility:private"],
     deps = if_static(
-        ["//tensorflow/compiler/xla/stream_executor/cuda:all_runtime"],
+        [":gpu_all_runtime"],
```

Comment:
That doesn't quite seem correct, is it saying to always use ROCM in OSS builds, and always use CUDA when building in google environment?

Reply:
@cheshire

Thanks for your comment.

I changed it to something that should be better.

PTAL.